server/core/src/config.rs

```diff
@@ -310,6 +334,8 @@ impl ServerConfigUntagged {
 #[derive(Debug, Deserialize, Default)]
 #[serde(deny_unknown_fields)]
 pub struct ServerConfigV2 {
+    #[allow(dead_code)]
+    version: String,
```

Comment:
I think you should add this to the 1.6.2 release notes ?

I hit this because it was a bit unexpected: 
```
kanidm        | Unable to parse config version from '"/data/server.toml"': TomlError { message: "data did not match any variant of untagged enum VersionDetection", raw: None, keys: [], span: None }
kanidm        | ERROR: Configuration Parse Failure: Custom { kind: InvalidData, error: TomlError { message: "data did not match any variant of untagged enum VersionDetection", raw: None, keys: [], span: None } }
```

Reply:
We have made a better error message here now, so I think we should be okay now.